marklogic-client-api/src/main/java/com/marklogic/client/datamovement/ZipSplitter.java

```diff
@@ -44,7 +40,6 @@
 public class ZipSplitter implements Splitter<BytesHandle> {
     private Map<String, Format> extensionFormats;
     private Predicate<ZipEntry> entryFilter;
-    private Function<String, String> uriTransformer;
```

Comment:
Given that we have already released ZipSplitter, it would be better to deprecate but not remove the existing uriTransformer with the following rules:

- During the deprecation period, people can use the old deprecated uriTransfomer or the new UriMaker but not both
- We call the old code path (old lines 329 through 334) or new code path (new lines 384 through 388) based on whether the old deprecated uriTransfomer or the new UriMaker is set

Reply:
UriTransfromer is back. Thanks.